.env

```diff
@@ -24,9 +24,7 @@ REACT_APP_INFURA_KEY=1221fd11e90849509afafd330ec7acc6
 REACT_APP_CHAIN_ID="1"
 REACT_APP_SUPPORTED_CHAIN_IDS="1,100,5"
 REACT_APP_NETWORK_URL_1=https://mainnet.infura.io/v3/1221fd11e90849509afafd330ec7acc6
-REACT_APP_NETWORK_URL_4=https://rinkeby.infura.io/v3/1221fd11e90849509afafd330ec7acc6
 REACT_APP_NETWORK_URL_5=https://goerli.infura.io/v3/1221fd11e90849509afafd330ec7acc6
-# REACT_APP_NETWORK_URL_100=https://rpc.gnosischain.com/oe-only
```

Comment:
not from this PR, but:
should we commit the .env file

Reply:
We already talked about this.
It's fine, and if you need to change anything locally you do on .env.local

.env is the default, while .env.production contains prod overwrites